src/components/DailyWeather/DailyWeatherDetails.tsx

```diff
@@ -2,17 +2,66 @@ import moment from "moment";
 import styled from "styled-components";
 import { ModalCloseButton } from "../Modal/ModalCloseButton";
 import { DailyWeatherProps, ItemDateInfo } from "./DailyWeather";
+import { DetailItem } from "./DetailItem";
+import { weatherZeroDescription } from "../../common/helpers";
+import { FormattedTemperature, FormattedTime } from "../ValueFormatters";
 
 const DailyWeatherDetailsWrapper = styled.div`
   position: relative;
   min-height: 5rem;
+
+  .temperature-value > span:first-child,
+  .time-value {
+    font-size: 1.2em;
+  }
+
+  .detail-item-wrapper {
+    border-bottom: 1px solid rgba(80, 80, 80, 0.2);
+  }
+`;
+
+const WeatherDescription = styled.div`
+  font-size: 0.8em;
+  display: inline;
+  vertical-align: bottom;
+`;
+
+const Spacer = styled.div`
+  width: 100%;
+  height: 1rem;
+`;
+
+const DetailValue = styled.span`
+  font-size: 1.2em;
 `;
 
 export const DailyWeatherDetails = ({ setSelectedDay, forecast }: DailyWeatherProps): JSX.Element => {
   return (
     <DailyWeatherDetailsWrapper>
-      <ItemDateInfo>{moment(forecast.dt * 1000).format("dddd, DD.M")}</ItemDateInfo>
+      <ItemDateInfo fontSize={"1.6rem"}>{moment(forecast.dt * 1000).format("dddd, DD.M")}</ItemDateInfo>
+      <Spacer />
       <ModalCloseButton onClick={() => setSelectedDay(null)} />
+      <DetailItem label={"weather"}>
+        <WeatherDescription>{weatherZeroDescription(forecast.weather)}</WeatherDescription>
```

Comment:
Optional - define `forecastWeatherZeroDescription`, looks like something that will be used in other places too.